packages/lodestar/src/chain/chain.ts

```diff
@@ -136,6 +138,11 @@ export class BeaconChain implements IBeaconChain {
       opts,
     });
 
+    this.eth2Context = {
```

Comment:
Don't attach this data to the state, it's unnecessary. Just pass the Eth2Context down to gossipsub